rpm-testing/README.md

```diff
@@ -9,13 +9,18 @@ that needs to be previously built or our official repository.
 
 Provision the Vagrant box using our official repository:
 
-```
-$ vagrant up
-```
+    $ vagrant up
 
 Alternatively, provision the box using the local repository (`../rpm`), which
 needs to be previously built:
 
-```
-$ REPOSITORY_CONFIG_FILE="archivematica-local.repo" vagrant up
-```
+    $ LOCAL_REPOSITORY="yes" vagrant up
+
+The script that provisions the VM prints the IP address, e.g.:
```

Comment:
We use fixed ips for our other vangrant envs ( https://github.com/artefactual/deploy-pub/blob/master/playbooks/archivematica-centos7/Vagrantfile#L11 ), We should do the same here, so the user knows where to look in advance,

Reply:
See bug https://github.com/artefactual-labs/am-packbuild/issues/133.

Reply:
Aside of a fixed ip, we should use vbox private_network. I hope you understand that bridging vm's with the public network, is not always a good idea (think about environments where dhcp is restricted to known MAC's, or have other network security measures to avoid rogue clients)